.github/CONTRIBUTING.rst

```diff
@@ -45,13 +45,9 @@ data to the new models.
 Code Convention
 ---------------
 
-We follow the `Django Coding Style`_ and enforce it using `flake8`_.
-
-In general, if flake8 is happy with your code, you should be fine. To use
-``flake8`` to check your code, you can use the following command::
-
-   $ flake8 --max-complexity=24 --statistics --benchmark --ignore=E5,F4 <project_dir>/
+We follow the `Black Coding Style`_, and sort our imports with `isort`_. This
```

Comment:
We might want to add something about pre-commit since that's the tool that actually runs in CI and lint step.

Reply:
Yea, I want to cross link to the getting started document from here, and move this bit of information over to there.

However, until we host it, that can't be done, so I made a temporary change that doesn't include the complete details.

Reply:
Well ok. Sounds good.